share/OpenMS/examples/external_code/CMakeLists.txt

```diff
@@ -25,34 +25,35 @@ if (OpenMS_FOUND)
   # check if the variable containing the include directories is defined
   if(NOT OpenMS_INCLUDE_DIRECTORIES)
     set(_message "The variable \${OpenMS_INCLUDE_DIRECTORIES} is not defined.")
-    set(_message "${_message}This CMakeLists.txt and the build of OpenMS seem to incompatible.")
+    set(_message "${_message}This CMakeLists.txt and your build of OpenMS seem incompatible.")
     set(_message "${_message}Please use the latest version from the OpenMS release!")
     message(FATAL_ERROR ${_message})
   endif()
 
   ## include directories for OpenMS headers (and contrib)
-  # Note: If you want to link against a specific libirary contained
-  #       in the OpenMS package you should also list the include
-  #       corresponding directory here, e.g.,
+  # Note: If you want to link against a specific library contained
+  #       in the OpenMS package you should also list the
+  #       corresponding include variable here, e.g.,
   #       OpenMS_GUI -> ${OpenMS_GUI_INCLUDE_DIRECTORIES}
   include_directories(${OpenMS_INCLUDE_DIRECTORIES})
 
   ## append precompiler macros and compiler flags specific to OpenMS
   ## Warning: this could be harmful to your project. Check this if problems occur.
   ## Also, use this to add your own compiler flags, e.g. for OpenMP support.
   ## e.g. for Visual Studio use /openmp
-  ## set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OPENMS_ADDCXX_FLAGS} /openmp")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OPENMS_ADDCXX_FLAGS}")
-  add_definitions(${OPENMS_DEFINITIONS})
-
+  # set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OPENMS_ADDCXX_FLAGS} /openmp")
+  
+  add_definitions(/DBOOST_ALL_NO_LIB) ## disable auto-linking of boost libs (Boost tends to guess wrong lib names)
```

Comment:
yep. at least thats what their docs say. Might not work in very old CMake versions ...